src/displayapp/screens/Dice.cpp

```diff
@@ -0,0 +1,231 @@
+#include "Dice.h"
+
+#include "Screen.h"
+#include "Symbols.h"
+#include "lvgl/lvgl.h"
+
+using namespace Pinetime::Applications::Screens;
+const uint8_t Dice::diceSizes[] = {2, 3, 4, 6, 8, 12, 20, 100};
+
+static void num_dice_inc_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
```

Comment:
multiple `auto dice can be declared as auto* dice` warnings

Reply:
@Riksu9000 Fixed this in https://github.com/JF002/InfiniTime/pull/565/commits/75c7e280842f08b91be91de121d600ec5f87defd